packages/node/src/yargs.ts

```diff
@@ -105,6 +105,11 @@ export const yargsOptions = yargs(hideBin(process.argv))
       describe: 'Timeout for indexer sandbox to execute the mapping functions',
       type: 'number',
     },
+    'block-time': {
```

Comment:
I'm not sure why we need to make this configurable, all users are going to want the same value. 

It seems we just need to set a better default block time that suits Avalanche. 

I think the better solution is to change the health check if we are at the latest blocks and are waiting for new blocks.